c2corg_ui/static/js/auth/authservice.js

```diff
@@ -363,7 +374,9 @@ app.Authentication.prototype.needAuthorization = function(method, url) {
       url.indexOf('/users/mailinglists') !== -1 ||
       url.indexOf('/users/following') !== -1 ||
       url.indexOf('/profiles') !== -1 ||
-      url.indexOf('/personal-feed') !== -1) {
+      url.indexOf('/personal-feed') !== -1 ||
+      url.indexOf('/xreports/edit') !== -1 ||
+      url.indexOf('/xreports/') !== 1) {
```

Comment:
Are you sure of ``!== 1``? Shouldn't it be ``-1``?
If yes, ``/xreports/`` should also work for ``/xreports/edit``?